packages/frontend/graphql/src/graphql/workspace-invoices-count.gql

```diff
@@ -0,0 +1,5 @@
+query getWorkspaceInvoicesCount($workspaceId: String!) {
+  workspace(id: $workspaceId) {
+    invoiceCount
```

Comment:
use workspaceInvoices ?

Reply:
It looks like we don't really need this `getWorkspaceInvoicesCountQuery`, I will delete it.